packages/publish_npm.sh

```diff
@@ -45,7 +45,8 @@ read -p "Enter [yes] to publish [no] " shouldContinue
 shopt -s nocasematch
 if [[ ${shouldContinue} == "yes" ]]; then
     echo "Building vertica-nodejs.."
-    yarn
+    rm -rf ../node_modules
```

Comment:
I think it would be good to add a comment that this line ensures the purpose of cleaning is well understood.   You made a comment about ensuring we build with the latest typescript.   Was there a specific problem you were solving, or was this just a general purpose step to ensure we build from a known state every time we publish?

Reply:
This is a general purpose step to ensure we build from a known state. I'll add a comment.